uint8 ID
float64 x #x position in meters
float64 y #y position in meters
float64 o #orientation in +- pi radians away from "east"